compar.c

```diff
@@ -11,6 +11,7 @@
 
 #include "id.h"
 #include "internal.h"
+#include "internal/class.h"
```

Comment:
Is this line needed?
Seems nothing changes...

Reply:
This and the enum.c change was needed in the first commit, but is not needed any more.  I'll push a rebased version that removes these.